pyproject.toml

```diff
@@ -11,17 +11,12 @@ requires = [
 
 [tool.cibuildwheel]
 build-verbosity = 1
-test-requires = [
-    "pytest",
-    "pytest-cov",
-    "PyYAML",
-    "scipy",
-    "sparse@git+https://github.com/pydata/sparse@updated-llvm-nightly-test"
-]
+test-requires = ["pytest", "pytest-cov", "PyYAML", "scipy"]
+before-test = "pip install --no-deps sparse@git+https://github.com/pydata/sparse@updated-llvm-nightly-test"
 test-command = "SPARSE_BACKEND=MLIR pytest --pyargs sparse.mlir_backend"
 
 [tool.cibuildwheel.linux]
-build = "cp310-* cp311-* cp312-*"
+build = "cp310-* cp311-* cp312-* cp313-*"
```

Comment:
And this one? Have just one in `tool.cibuildwheel`.

Reply:
Done!